src/styles/blocks/page.scss

```diff
@@ -0,0 +1,4 @@
+.page {
+  margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), zero values should not include units. The `margin` property should be written as `margin: 0;` without the unit.